Libraries/src/Amazon.Lambda.CognitoEvents/Amazon.Lambda.CognitoEvents.csproj

```diff
@@ -6,7 +6,7 @@
     <Description>Amazon Lambda .NET Core support - CognitoEvents package.</Description>
     <TargetFrameworks>netstandard2.0;netcoreapp3.1;net8.0</TargetFrameworks>
```

Comment:
Unrelated to the actual change, but since we're doing a major version bump, should we just drop `netcoreapp3.1`?

Reply:
I think we should.